Description:  Do not run test suite threaded, does not work on all arches.
Author: Joachim Breitner <nomeata@debian.org>

--- haskell-attoparsec-0.10.1.1.orig/attoparsec.cabal
+++ haskell-attoparsec-0.10.1.1/attoparsec.cabal
@@ -74,7 +74,7 @@ test-suite tests
                   QC.Text
 
   ghc-options:
-    -Wall -threaded -rtsopts
+    -Wall -rtsopts
 
   build-depends:
     attoparsec,
